server/utils/bedUtils.test.ts

```diff
@@ -65,12 +67,32 @@ describe('bedUtils', () => {
   })
 
   describe('actionCell', () => {
+    describe('when the user has the FUTURE_MANAGER role', () => {
+      const user = userDetailsFactory.build({ roles: ['future_manager'] })
+
+      it('returns a "V2" link to manage the room', () => {
+        expect(actionCell(bed, premisesId, user)).toEqual({
+          html: v2BedLink(bed, premisesId),
+        })
+      })
+    })
+
     describe('when the user does NOT have the FUTURE_MANAGER role', () => {
       const user = userDetailsFactory.build({ roles: ['manager'] })
 
       it('returns a "V1" link to manage the room', () => {
         expect(actionCell(bed, premisesId, user)).toEqual({
-          html: bedLink(bed, premisesId),
+          html: v1BedLink(bed, premisesId),
+        })
+      })
+    })
+
+    describe('when no user is given', () => {
+      const nullUser: UserDetails = null
```

Comment:
If `actionCell` is only called with 2 arguments then the `user` param will be `undefined` (not `null`). There are some subtle differences in the behaviour between  the`null` and `undefined` primitives so I think it would be better to use `undefined` here just to be safe.

Reply:
thanks Rich, I've updated this to describe the behaviour with `undefined` rather than `null`